src/user/relationship-model.md

```diff
@@ -71,3 +71,15 @@ One important thing to note about bidirectional relationships, are that the
 "from" and "to" sides are still significant in the database, meaning each entity
 must match the constraint for that side. However, from a user perspective, which
 side each entity is stored as is insignificant.
+
+## Example { #relationship_model_example }
```

Comment:
Not sure how exactly but it would be great if that image would integrate with all of the text above. I think that would make the text easier to understand. Also an image where the relationships are nodes with from/to edges would help.